config.json

```diff
@@ -1,7 +1,18 @@
 {
   "scp-wiki": {
     "id": 66711,
-    "index": [],
+    "index": [
+      "scp-series",
+      "scp-series-2",
+      "scp-series-3",
+      "scp-series-4",
+      "scp-series-5",
+      "scp-series-6",
+      "joke-scps",
+      "scp-ex",
+      "archived-scps",
+      "attribution-metadata"
```

Comment:
For the moment, Sherlock does not support data via "attribution-metadata". It relies on Wikidot's AJAX API to find the author and date of publication. Maybe in the next release :)